### PR TITLE
fix #72816 desishoot.fun

### LINKED ISSUE #72816: desishoot.fun
### Issue URL (Incorrect Blocking)
[NSFW] [https://desishoot.fun](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fdesishoot.fun)
### Comment
Username: @tricksnew
> 
> --
> The information below is automatically generated. Do not delete it.
> 
> Referrer:
> https://desishoot.fun/category/desi-pics/desi-nude-pics/page/2
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### [NSFW] Screenshots
<details>
  <summary>Screenshot 1</summary>

  [Screenshot 1](https://reports-img.adguard.com/kM3BLHI.png)
</details>
<details>
  <summary>Screenshot 2</summary>

  [Screenshot 2](https://reports-img.adguard.com/Sh5C8Jk.png)
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Firefox
AdGuard driver: | TDI
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Self-destructing first-party cookies (4320), <br/>Self-destructing third-party cookies (180), <br/>Block Push API, <br/>Block Location API
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances
Other extensions used: | i dont use other but adguard is not working in this site it saying adguard detected

### PR DESCRIPTION
**Pull request by Yuki2718**
https://github.com/AdguardTeam/AdguardFilters/issues/72816

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -254,6 +254,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+desishoot.fun##.islemag-content-right > .widget_custom_html
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:257]: Missed tag after `.islemag-content-right > `

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -35,6 +35,7 @@ klasizle.com,ugurfilm.com,bestdizi.com,filmmom.net,netfullfilmizle.com,filmyani.
...
 !
+desishoot.fun###chaty-channel-telegram
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:38]: It's a floating element, please move it to the Annoyances -> popups

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (Yuki2718) [reply EnglishFilter/sections/specific.txt:257]: Okay, will add.

Comment (Yuki2718) [reply EnglishFilter/sections/specific.txt:257]: `.islemag-content-right` doesn't need tag?

Comment (zloyden) [reply EnglishFilter/sections/specific.txt]: That's not necessary for first element, we add tags after `>`.
Also we skip it, if element has different tags.
So, the rule becomes more accurate and convenient for our team to review.

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (Yuki2718) [reply EnglishFilter/sections/specific.txt]: I got it, and the same goes for descendant combinator (space)?

Comment (zloyden) [reply EnglishFilter/sections/specific.txt]: Yes, the same)

**Pull request merged**